components/Login/index.tsx

```diff
@@ -0,0 +1,57 @@
+import Link from "next/link";
+import { Field, Formik } from "formik";
+import { FC } from "react";
+import {
+  Button,
+  Container,
+  FormBlock,
+  Header,
+  Heading,
+  Input,
+  Links,
+  Wrapper,
+} from "../../styles/authpage";
+import { WebsiteUrls } from "../../types/enums";
+import { Values } from "../../types/formik";
+
+const Login: FC = () => {
```

Comment:
`Login` and `Registration` contain a lot of duplicated code. Extract common parts